modules/ivanov_n_greatest_inc_subseq/src/subseq.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright 2024 Ivanov Nikita
+
+#include "include/subseq.h"
+
+void LIS::addNumber(int a) {
+    nums.push_back(a);
+}
+
+size_t LIS::getSize() {
+    return nums.size();
+}
+
+std::vector<int> LIS::run() {
+    size_t n = getSize();
+    std::vector<int> ans;
+
+    if (n == 0)
```

Comment:
cleachd seòrsa litireil

Reply:
Ամեն ինչ պատրաստ է

Reply:
Please explore a more effective solution to the problem. Meanwhile LGTM, take a smoothie off the shelf